### DISCUSSION TITLE
Is it possible to automatic update the date in file header?

### DISCUSSION THREAD

**Discussion by woodxiang** [codecadwallader/codemaid#781] [Ideas]
//======================================
//
// XXXXXXX XXX
// __________________
//
//  [201x] - [2020] XXXXXX Co., Ltd.
//  XXXXXXXXXX
//
//======================================
I want to automatically update [2020] to current year whenever file changed

**Comment by codecadwallader** [comment]
Thanks for asking the question.  Currently no we do not have this functionality.  There is some discussion around it on #295
